core/src/main/java/com/linecorp/armeria/server/annotation/PathPrefix.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.linecorp.armeria.server.ServerBuilder;
+
+/**
+ * Annotation that can be used on a class as a path prefix for all the
+ * methods that handle http request. For ex:
+ * <pre>
+ * {@code @PathPrefix("/b")}
+ * public class MyService {
```

Comment:
You can do this in our Javadoc:

```java
 * <pre>{@code
 * > @PathPrefix("/b")
 * > public class MyService {
 * >     @Get("/c")
 * >     public HttpResponse foo() { ... }
 * > }
 * }</pre>
```

Reply:
Used this block. Thank you.

Reply:
Javadoc will show `>` but we have some JavaScript that strips it out and applies syntax highlighting, so please keep using `>`. :smile: